src/lib.rs

```diff
@@ -106,7 +108,18 @@ fn collect_span(span: &DiagnosticSpan) -> Option<Replacement> {
     })
 }
 
-pub fn collect_suggestions(diagnostic: &Diagnostic) -> Option<Suggestion> {
+pub fn collect_suggestions(diagnostic: &Diagnostic, only: &Option<HashSet<String>>) -> Option<Suggestion> {
```

Comment:
Hm, what's better: Using an Option `if let` or an empty HashSet with `is_empty()`?

Reply:
yea, the latter might be easier